plugins/woocommerce/includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1528,7 +1529,7 @@ public function get_wizard_in_cart_payment_gateways() {
 		$can_paypal  = $this->is_paypal_supported_currency( $currency );
 
 		if ( ! current_user_can( 'install_plugins' ) ) {
-			return $can_paypal ? array( 'paypal' => $gateways['paypal'] ) : array();
+			return $can_paypal ? array( PaymentMethods::PAYPAL => $gateways['paypal'] ) : array();
```

Comment:
```suggestion
			return $can_paypal ? array( PaymentMethods::PAYPAL => $gateways[PaymentMethods::PAYPAL] ) : array();
```

I might be mistaken, but shouldn't we use the constant twice in this case?